sort_images.sh

```diff
@@ -0,0 +1,159 @@
+#! /usr/bin/env bash
+
+#----------------------------------------------------------------------
+# NAME: sort_images.sh
+#
+# BRIEF: Command-line Bash script with prompts to help sort photo files.
+# - creates & sorts image files into year & month subdirectories
+# - also creates day subdirectories, if answer at prompt is `Y` for Yes
+# - sorts by photo-taken date or if missing, by modify date - since sometimes
+# creation date is assigned as file download date, which is incorrect.
+#
+# AUTHOR: Kim Lew
+#----------------------------------------------------------------------
+
+if [ $# -eq 0 ]; then
+  echo "Type the directory path that needs Year & Month subdirectories."
+  echo "Leave off trailing / at end of path: "
+  read -r directory_path
+  echo "Do you also want a Day subdirectory? (Y for Yes, N for No):"
+  read -r day_subdir_also
+else
+  if [ ! "$1" ]; then
+    echo "Enter the directory path - as the 1st parameter."
+    echo "Enter Y for Yes or N for No - for Day subdirectory as 2nd parameter."
+    exit 1
+  fi
+fi
+
+if [[ $# -eq 1 || $# -eq 2 ]]; then
+  directory_path="$1"
+  safe_day_subdir_also='n'
+
+  if [ $# -eq 2 ]; then
+    day_subdir_also="$2"
+
+    case "$day_subdir_also" in
```

Comment:
For consistency you probably mean to use curly braces `"${day_subdir_also}"`

Reply:
@kcharter Rats! I missed that one. Thanks. Changed.